main.go

```diff
@@ -142,6 +156,8 @@ func main() {
 			RunScraper(conf)
 		} else if *mode == "webapp" {
 			RunServer(conf)
+		} else if *mode == "filesize_fetcher" {
```

Comment:
Yeah, but for now it only updates the filesize, so I think it's more appropriate calling it like this. But I can rename it if you want to.